Jenkinsfile.cloud

```diff
@@ -131,21 +131,34 @@ node(NODE) {
         sh "qemu-img convert -f raw -O qcow2 ${image} ${qcow}"
         sh "rm -f ${image}"
     }
+    // These three are generated from the QCOW2
     par_stages["vmdk"] = { -> stage("Generate vmdk") {
         sh """env LIBGUESTFS_BACKEND=direct ${WORKSPACE}/scripts/coreos-oemid ${qcow} ${vmdk}.tmp vmware
               qemu-img convert -f qcow2 -O vmdk ${vmdk}.tmp -o adapter_type=lsilogic,subformat=streamOptimized,compat6 ${vmdk}
               rm ${vmdk}.tmp"""
     } }
+    // But note that EC2 goes into workspace, we upload it directly
+    ec2 = "${WORKSPACE}/rhcos-aws-${commit}.vmdk"
+    par_stages["ec2"] = { -> stage("Generate EC2") {
+        sh """env LIBGUESTFS_BACKEND=direct ${WORKSPACE}/scripts/coreos-oemid ${qcow} ${WORKSPACE}/rhcos-aws.qcow2 ec2
+              qemu-img convert -f qcow2 -O vmdk ${WORKSPACE}/rhcos-aws.qcow2 -o adapter_type=lsilogic,subformat=streamOptimized,compat6 ${ec2}
+              rm ${WORKSPACE}/rhcos-aws.qcow2"""
+    } }
     par_stages["vagrant-libvirt"] = { -> stage("Generate vagrant-libvirt") {
         // We use direct as we hit SELinux issues otherwise
         sh "env LIBGUESTFS_BACKEND=direct ${WORKSPACE}/qcow2-to-vagrant/qcow2-to-vagrant ${qcow} ${vagrant_libvirt}"
     } }
+    // Execute parallel group
     parallel par_stages; par_stages = [:]
 
-    stage("Finalizing, generate metadata") {
+    // Convert the qcow2 to have an openstack stamp, and gzip
+    stage("OpenStack qcow2/ec2") {
```

Comment:
Even though we're not actually running `qemu-img convert`, let's just call this `Generate openstack` to be consistent?